1.20.5/1.20.5-pre4.md

```diff
@@ -0,0 +1,3 @@
+pack breaking | The Resource Pack version is now 32
+
+font | The `shift` in TTF glyph providers is now restricted to the range `[-512; 512]`
```

Comment:
Should have tags `assets`, `font`, and `breaking`.

Reply:
fixed